mdnsd.c

```diff
@@ -161,6 +161,9 @@ int main(int argc, char *argv[])
 				printf("can't write to socket: %s\n", strerror(errno));
 				return 1;
 			}
+			else {
```

Comment:
Minor nitpick, the coding style is Linux, so else's should be cuddled, like this `} else {`. But I'm gonna merge anyway and fix later :)